aiobungie/internal/factory.py

```diff
@@ -387,7 +387,7 @@ def _set_character_attrs(self, payload: typedefs.JSONObject) -> character.Charac
             gender=enums.Gender(payload["genderType"]),
             race=enums.Race(payload["raceType"]),
             class_type=enums.Class(payload["classType"]),
-            emblem=assets.Image(str(payload["emblemBackgroundPath"])),
+            emblem=assets.Image(str(payload.get("emblemBackgroundPath"))),
```

Comment:
If This line `str(payload.get(...))` returns `None` then the image repr will be like this `Image("None")` which's incorrect.
Just remove the `str()` call

```suggestion
            emblem=assets.Image(payload.get("emblemBackgroundPath")),
```